ivy/functional/frontends/tensorflow/signal.py

```diff
@@ -12,3 +12,15 @@ def kaiser_window(window_length, beta=12.0, dtype=ivy.float32, name=None):
 
 
 kaiser_window.supported_dtypes = ("float32", "float64", "float16", "bfloat16")
+
+
+@handle_tf_dtype
+@to_ivy_arrays_and_back
+def stft(signal, *, frame_length: Union[int, ivy.Array, ivy.NativeArray],
```

Comment:
You shouldn't use the same function formatting and type hints here that ivy functions use. The frontends' type hints should exactly match the type hints of the native function itself.
Therefore, in this case, the `stft's`  signature should be the same as `tf.signal.stft`